### DISCUSSION TITLE
It seems like the 'only_active' API parameter for getting the DAG list in Airflow is not functioning

### DISCUSSION THREAD

**Discussion by HwiLu** [apache/airflow#34384] [Q&A] [answered]
I'm using the following API interface, specifying 'only_active' as 'true,' but I'm still getting the entire DAG list, including paused ones, instead of just getting the DAGs with an 'active' status. Can someone please tell me if I'm using it incorrectly or if this is a bug in the Airflow API? I've browsed through the issues, but I couldn't find any relevant information. I used Swagger for api test.

`curl -X 'GET' \
  'http://192.168.xx.xx:8080/api/v1/dags?limit=100&only_active=true' \
  -H 'accept: application/json'`

**Comment by potiuk** [comment, marked as answer]
Not sure which version of Airflow you have, but regardless active has nothing to do with paused.

* Active is when the dag is currently present in DAGs folder. When you delete DAG from DAGs folder, it's still present in the database, but it is in "inactive" state (when you add it back it will become active again).
* Paused is when you manually pause an active DAG for execution

When you get DAG information you can see "is_paused" to indicate if the DAG is paused.

The "paused" filter in the API has been added in 2.6.0 - so if you want to use it and you have an earlier version of Airlfow you will need to upgrade https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/get_dags

<img width="1049" alt="Screenshot 2023-09-15 at 09 14 07" src="https://github.com/apache/airflow/assets/595491/1b248f94-628b-4702-9084-73504f35a53c">

**Reply by HwiLu** [reply]
My airflow version is 2.5.2
Thank you for your response. You've answered a question that has been bothering me for a long time. I used to think that 'is_active' in the API represented the 'active' list on the Airflow homepage's DAG list. I have now achieved this by fetching all the DAGs and then filtering out the ones where 'is_paused' is false.
